bin/perseo

```diff
@@ -80,6 +80,25 @@ function loadConfiguration() {
     if (process.env.PERSEO_LOG_LEVEL) {
         config.logLevel = process.env.PERSEO_LOG_LEVEL;
     }
+
+    config.smtp = config.smtp || {};
+    config.smtp.port = process.env.PERSEO_SMTP_PORT || config.smtp.port;
```

Comment:
I think it would be a good idea to mention in the documentation which parameter can be passed as env variables (and the nambe of the variable used in each case).

(It there are already mentioned in documentation, please ignore this comment).